src/main/com/yetanalytics/lrs/xapi/document.cljc

```diff
@@ -17,11 +17,12 @@
   (timestamp/stamp-now))
 
 (s/def ::content-type
-  string-ascii-not-empty)
+  (s/nilable string?))
```

Comment:
What happened to `string-ascii-not-empty`?

Reply:
oh right, I guess it has a nice gen on there